dockerfiles/Dockerfile.ruby-maze-runner

```diff
@@ -20,7 +20,7 @@ COPY spec ./spec
 RUN gem build bugsnag.gemspec
 
 # The maze-runner node tests
-FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:latest-cli as ruby-maze-runner
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:v3.0.2-cli as ruby-maze-runner
```

Comment:
Yeah, fine to pin to a version, but I would request that you keep an eye out for new releases and consider upgrading occasionally as not to be left too far behind.

Reply:
Yeah, I just want to avoid the same thing happening again; this was quite painful to debug, especially with Rubocop failing at the same time :s

Reply:
👍  Once bitten...